README.md

```diff
@@ -293,8 +293,8 @@ git commit -m "Updating version in preparation for a new release"
 
 # install fontbakery on it and run our code tests
 pip install .
```

Comment:
you don't need to install fontbakery to run its test suite, that's done automatically by tox inside the temporary virtual environents that it creates.

Reply:
Oops, forgot to delete that...